sentinel/tests/unit/lib/feed.js

```diff
@@ -1,7 +1,8 @@
 const sinon = require('sinon');
 const chai = require('chai');
 const db = require('../../../src/db');
-const feed = require('../../../src/lib/feed');
+const feed = require('../../../src/lib/feed'),
+      tombstoneUtils = require('@medic/tombstone-utils');
```

Comment:
I think I prefer having each import with its own `const`. Mainly this is so the diff is cleaner - you don't have to modify the line above to add a new one. I don't care enough to raise it, except all the lines above do this, so now the new one stands out!

Reply:
Done!